src/modules/DepositionGeant4/GeneratorActionG4.cpp

```diff
@@ -350,6 +350,13 @@ void GeneratorActionG4::GeneratePrimaries(G4Event* event) {
         }
     }
 
+    // Set the time of the particle source within the time window
+    if(time_window_ > 0) {
+        auto* single_source = particle_source_->GetCurrentSource();
+        double event_time = time_ + G4UniformRand() * time_window_;
+        single_source->SetParticleTime(event_time);
+    }
```

Comment:
We should probably explicitly set it to `0.0` otherwise since I don't know what it is initialized to - and would not rely on G4 not changing this..

Reply:
This should have been set to `time_` when initialised which is by default `0.0`.

The `make format` command is very nice.